Makefile

```diff
@@ -416,20 +415,18 @@ update: ## update dependencies
 	uv sync --upgrade
 
 publish_base:
-	#TODO: replace with uvx twine upload dist/*
-	cd src/backend/base && poetry publish --skip-existing
+	cd src/backend/base && uv publish
 
 publish_langflow:
-	#TODO: replace with uvx twine upload dist/*
-	poetry publish
+	uv publish
 
 publish_base_testpypi:
-	#TODO: replace with uvx twine upload dist/*
-	cd src/backend/base && poetry publish --skip-existing -r test-pypi
+	# TODO: update this to use the test-pypi repository
+	cd src/backend/base && uv publish -r test-pypi
 
 publish_langflow_testpypi:
-	#TODO: replace with uvx twine upload dist/*
-	poetry publish -r test-pypi
+	# TODO: update this to use the test-pypi repository
+	uv publish -r test-pypi
```

Comment:
I don't believe `uv` supports the `-r` flag yet. Here's a recent (somewhat related) issue about that - https://github.com/astral-sh/uv/issues/7676. 

Maybe need to use `--publish-url`? For now, we can just remove these or comment so as to not confuse others.